store/keystore.go

```diff
@@ -199,3 +211,36 @@ func saveToPem(out io.Writer, key openssl.PrivateKey) error {
 
 	return err
 }
+
+func loadPassphrase(passphrase_file string) (passphrase string, err error) {
```

Comment:
I see now that the test suite is complaining about code coverage. Could you also add a unit test for this function? It's not well covered at the moment.

The stdin case is probably going to be tricky to test, so we can skip that, but at least the "from file" codepath should be tested, with some dummy file created in the test code.

Reply:
Also saw the coverage error and already working on this.